catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ScoreCommand.java

```diff
@@ -0,0 +1,253 @@
+/**
```

Comment:
I'm going to switch this one to solely work on metacards, rather than files. I think that makes more sense for where we eventually want this to go. Also, if we want it to work on both, I can abstract the functionality.